com.unity.visualeffectgraph/Editor/GraphView/VFXViewWindow.cs

```diff
@@ -80,8 +83,11 @@ public void LoadResource(VisualEffectResource resource, VisualEffect effectToAtt
             {
                 InternalLoadResource(resource);
             }
-            if (effectToAttach != null && graphView.controller != null && graphView.controller.model != null && effectToAttach.visualEffectAsset == graphView.controller.model.asset)
-                graphView.attachedComponent = effectToAttach;
+
+            if (!graphView.TryAttachTo(effectToAttach))
+            {
+                s_LastAttachedComponent = null;
```

Comment:
The introduction of this static variable was done to fix a bug when "unmaximizing" the window. Because in that process the window is recreated and then losing the attachment.